protostar/commands/test/testing_live_logger.py

```diff
@@ -47,8 +99,9 @@ def log(
         self,
         shared_tests_state: SharedTestsState,
         test_collector_result: "TestCollector.Result",
+        structured_format: bool,
+        messanger: Optional[Messenger] = None,
```

Comment:
```suggestion
        messanger: Messenger,
```
Messenger should always be required.